package.json

```diff
@@ -17,7 +17,6 @@
     "glob-parent": "^5.1.2",
     "json-schema-ref-parser": "^9.0.9",
     "jsonschema": "^1.4.0",
-    "lzutf8": "0.6.0",
```

Comment:
we would need lz-string as dependency in fca right / since we are removing lzutf8 ? will standalone work

Reply:
removed because lzutf8 are not using anywhere in fca